images/applications/ts-ps.bash

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
 # Dipole
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS02:CO-PSCtrl-TS1 &> /ioc-logs/sirius-ioc-ts-ps-dipoles &
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS02:CO-PSCtrl-TS1 &> /ioc-logs/sirius-ioc-ts-ps-dipoles & # 10.0%
 sleep 2 # before starting other IOCs
 
 # Quadrupoles
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS02:CO-PSCtrl-TS2 &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-12 &
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS04:CO-PSCtrl-TS &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-34 &
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS02:CO-PSCtrl-TS2 &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-12 & # 13.6%
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaPS04:CO-PSCtrl-TS &> /ioc-logs/sirius-ioc-ts-ps-quadrupoles-34 & # 13.9%
 
 # Correctors
-/usr/local/bin/sirius-ioc-as-ps.py LA-RaCtrl:CO-PSCtrl-TS &> /ioc-logs/env-sirius-ioc-ts-ps-correctors &
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaCtrl:CO-PSCtrl-TS &> /ioc-logs/env-sirius-ioc-ts-ps-correctors & # 40.2%
```

Comment:
Is this log file correct? wouldn't it be `sirius-ioc-ts-ps-correctors`?

Reply:
it is. will be fixed in my next commit